### PR TITLE
Update de.yml

### PR DESCRIPTION
**Pull request by SuriyaaKudoIsc**
@saivann @harding @wbinns @gavinandresen @luke-jr:

Hi guys, :smile:

I **improved the translation formattings for `de.yml`**.

*Yours,*
[**Suriyaa Kudo**](https://github.com/SuriyaaKudoIsc) :octocat:

### CODE REVIEW

File: _translations/de.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ de:
     title: "Über bitcoin.org"
     pagetitle: "Über bitcoin.org"
     pagedesc: "Bitcoin.org ist einer nachhaltigen Entwicklung von Bitcoin gewidmet."
-    own: "Wem gehört bitcoin.org?"
+    own: "Wem gehört eigentlich bitcoin.org?"
```
Comment (MarcoFalke) [inline _translations/de.yml]: I don't think we need a Füllwort here.

Comment (jonasschnelli) [inline _translations/de.yml]: Agree with @MarcoFalke

**Comment by SuriyaaKudoIsc** [comment]
@MarcoFalke: Ok. :+1:

File: _translations/de.yml (hunk 1 above)
Comment (SuriyaaKudoIsc) [inline _translations/de.yml]: @MarcoFalke: Ok. :+1:

**Comment by SuriyaaKudoIsc** [comment]
@MarcoFalke & @jonasschnelli: This PR can be closed.

**Pull request closed without merging**